modules/041-linstor/templates/linstor-csi/linstorcsidriver.yaml

```diff
@@ -170,9 +170,23 @@ spec:
       maxUnavailable: 1
     {{- end }}
   controllerEndpoint: https://linstor.d8-{{ .Chart.Name }}.svc:3371
+  {{- if and (.Values.linstor.dataNodes) (.Values.linstor.dataNodes.nodeSelector) }}
```

Comment:
It will fail if there isnt a value `.Values.linstor.dataNodes.nodeSelector`. AFAIK.

Reply:
Need two if'es.